rubocop/.rubocop-common.yml

```diff
@@ -17,6 +17,15 @@ Layout/CaseIndentation:
 Layout/FirstArrayElementIndentation:
   EnforcedStyle: consistent
 
+Layout/FirstArrayElementLineBreak:
+  Enabled: true
+
+Layout/FirstHashElementLineBreak:
+  Enabled: true
+
+Layout/FirstMethodArgumentLineBreak:
```

Comment:
I like these but I did find on one of my projects, this rule's autocorrect is a little goofy:

```
     def initialize(attrs)
-      super(*attrs.values_at(:id, :name, :street, :city, :state,
-                             :cross_street, :phone, :zipcode, :profile_image, :lat, :lng))
+      super(
+*attrs.values_at(
+  :id,
+  :name,
+  :street,
+  :city,
+  :state,
+  :cross_street,
+  :phone,
+  :zipcode,
+  :profile_image,
+  :lat,
+  :lng,
+))
     end
```

Granted - this method call in the first place is a little goofy but the autocorrect took all the indentation out.  Just FYI.  I'm going to put the other 5 in my side project tho.  Like @dylnclrk I also have done the 
```
   delegate :this, :that, :thus,
     :whatever, :here, to: :thing
```
and have never really been happy with it.  But I'm happy to let the computer just make them all consistent.

Reply:
It seems to be a problem specific to `super`. So I think the auto-correct behavior is actually a bug in rubocop. I'll report it.

Reply:
@bunnymatic FYI the weird indentation autocorrect in your example was a bug and should be fixed in the next rubocop release: https://github.com/rubocop/rubocop/pull/11081

Reply:
Nice.  Thanks